traffic_ops/v3-client/topology.go

```diff
@@ -23,12 +23,15 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
+// ApiTopologies is Deprecated: will be removed in the next major version. Be aware this may not be the URI being requested, for clients created with Login and ClientOps.ForceLatestAPI false.
 const ApiTopologies = apiBase + "/topologies"
 
+const APITopologies = "/topologies"
```

Comment:
I had the same thought. I could imagine it being convenient and letting someone avoid duplicating, if they needed to get a route but not deserialize it with the standard mechanisms for some reason.

But since we're changing symbols anyway, it's not a deprecation issue, and we can always add them back later. I don't object to making them all private for now, if that's the consensus.

On that subject, I think there's a chance ORT may want or need to get objects without serialization at some point. As we move toward breaking up ORT and making it UNIX-style, we'll likely end up passing objects between apps. Parsing and serializing JSON is incredibly expensive, and it may end up making ORT take seconds or even minutes longer to run, if we don't have a way to request objects from TO without parsing them. Something to keep in mind.

Reply:
Ok, I do like keeping the surface area of public symbols to a minimum, but I don't care too strongly about this.